docs/pages/kubernetes-access/helm/reference/teleport-kube-agent.mdx

```diff
@@ -192,6 +192,48 @@ You can specify multiple apps by adding additional list elements.
   You can see a list of all the supported [values which can be used in a Teleport application access configuration here](../../../application-access/reference.mdx#configuration).
 </Admonition>
 
+## `appResources`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `list` | `[]` | When `app` chart role is used at least one of `apps` and `appResources` is required. |
+
+`appResources` is a YAML list object detailing the resource selectors of the applications that should be proxied by Teleport Application access.
+
+You can specify multiple selectors by adding additional list elements.
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  appResources:
+  - labels:
+      "env": "prod"
+  - labels:
+      "env": "test"
+  ```
+
+  <Admonition type="note" title="YAML formatting">
+    YAML is very sensitive to correct spacing. When specifying lists in a `values.yaml` file, you might like
+    to [use a linter](https://codebeautify.org/yaml-validator) to validate your YAML list and ensure that it is correctly formatted.
```

Comment:
I feel like this should be present everywhere we have a yaml snippet :sweat_smile: